json-endpoints/class.wpcom-json-api-list-roles-endpoint.php

```diff
@@ -130,6 +130,8 @@ function callback( $path = '', $blog_id = 0 ) {
 		// Sort the array so roles with the most number of capabilities comes first, then the next role, and so on
 		usort( $roles, array( 'self', 'role_sort' ) );
 
+		$roles = apply_filters( 'wpcom_api_site_roles', $roles );
```

Comment:
Could you add a docblock to that new filter, so a new Codex page can be automatically created by our parser once this gets released? Here is an example:
https://github.com/Automattic/jetpack/blob/99ec0afb51d8b2e5e439e4e3451225d183b37abf/json-endpoints/class.wpcom-json-api-update-comment-endpoint.php#L179-L188